cdap-docs/admin-manual/source/installation/installation.rst

```diff
@@ -523,6 +531,41 @@ For instructions on enabling CDAP Security, see :doc:`CDAP Security <security>`;
 and in particular, see the instructions for 
 :ref:`configuring the properties of cdap-site.xml <enabling-security>`.
 
+Configuring Hortonworks Data Platform
+.....................................
+Beginning with `Hortonworks Data Platform (HDP) 2.2 <http://hortonworks.com>`__, the
```

Comment:
This line doesn't make sense to me. Are the mapreduce jars stored in HDFS instead of local file system? @wolf31o2  can you clarify?